SH4RE/SH4RE/Account/MyListingsView.swift

```diff
@@ -35,8 +42,11 @@ struct MyListingsView: View {
         }
         .navigationTitle("My Listings")
         .onAppear(){
-            // TODO: change this to fetch current user listings (not all)
             fetchUsersListings(uid: getCurrentUserUid(), completion: { listings in
+                if (listings.isEmpty) {
+                    self.listingsView.listings.removeAll()
```

Comment:
honestly can't explain this one, I just noticed that when `listings` was empty the `self.listingsView.listings` still had the last item left over, so I have to explicitly remove all from the array rather than assign it an empty array. It must have something to do with how Swift handles emptying array